quickstarts/microsoft.web/app-service-managed-certificate-apex/azuredeploy.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "SubjectName": {
+            "type": "string",
+            "metadata": {
+                "description": "Subject name of the certificate."
+            }
+        },
+        "AppServicePlanName": {
+            "type": "string",
+            "metadata": {
+                "description": "App Service Plan Name where certificate will be imported to."
+            }
+        },
+        "Location": {
+            "type": "string",
+            "metadata": {
+                "description": "Location of app service plan."
+            }
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2019-08-01",
+            "location": "[parameters('Location')]",
+            "name": "[concat(parameters('SubjectName'), '-', parameters('AppServicePlanName'), '-', parameters('Location'))]",
+            "type": "Microsoft.Web/certificates", 
+            "properties": {
+                "serverFarmId": "[resourceId('Microsoft.Web/serverfarms/', parameters('AppServicePlanName'))]",
```

Comment:
the resourceGroup parameter needs to be added here (and as a param to the template)

Reply:
as on the properties we are informing the serverFarmId, the API will take care of the Resource Group as it should match the app service plan

Reply:
only if they are deployed to the same resourceGroup - which does not have to be the case.